numpy/ma/core.py

```diff
@@ -786,22 +786,17 @@ def fix_invalid(a, mask=nomask, copy=True, fill_value=None):
 
 class _DomainCheckInterval(object):
     """
-    Define a valid interval, so that :
-
-    ``domain_check_interval(a,b)(x) == True`` where
-    ``x < a`` or ``x > b``.
+    Define a valid interval, so that:
 
+    ``_DomainCheckInterval(a,b)(x)`` is False where ``a <= x <= b``.
```

Comment:
Not sure this is really the same as the old comment because of nans. The old comment more literally describes which ufuncs are used in `__call__` (`less`, `logical_or`, `greater`).

That is, `x > a` is not the opposite of `a <= x` for `x = np.nan`.